packages/discussions/lib/Comment/index.js

```diff
@@ -1,3 +1,22 @@
+const featuredTargets = [
+  { key: 'DEFAULT', default: true },
+  { key: 'MARKETING' }
+]
+
+if (!featuredTargets.find((t) => t.default)) {
+  throw new Error('At least featuredTarget needs default flag')
```

Comment:
Nitpicking for clarity's sake
```suggestion
  throw new Error('At least one featuredTarget needs default flag')
```